views/failure_to_proof.erb

```diff
@@ -0,0 +1,8 @@
+<div class="clearfix">
+  <div class="col-12 sm-col-6 mx-auto">
+    <div class="h5 p2 center">
+      <span class="bold caps">Failure to Proof</span>
+      <p><strong>We were unable to verify your identity</strong>.</p>
+    </div>
+  </div>
+</div>
```

Comment:
Could we use the same markup as in the OIDC implementation in https://github.com/18F/identity-oidc-sinatra/pull/141 ? Not sure if this is meant to be internally consistent with other SAML views, but I think it'd make sense for the two sample apps to look the same, and these BassCSS classes (`mx-auto`, etc.) are legacy and undesirable vs. design system.

```suggestion
<h1>Failure to Proof</h1>
<div>We were unable to verify your identity.</div>
```

Reply:
hi @aduth  ... yes, I was attempting to be consistent with the other saml views.  I am also good with your suggestion to be consistent witht the oidc app.  accepted ... thanks!
<img width="400" alt="Screen Shot 2023-04-28 at 11 20 26 AM" src="https://user-images.githubusercontent.com/1261794/235188168-28637c24-8a00-4d49-8f27-59740f0de38d.png">